partials/tabs/about.html

```diff
@@ -22,8 +22,8 @@
                languages supported and the bears themselves, click here.</p>
           </div>
           <div class="card-action">
-            <a href="#">Bears Documentation</a>
-            <a href="#">Bears Source</a>
+            <a href="https://docs.coala.io/en/latest/">Documentation</a>
```

Comment:
This shall still point to the bears doc right? Is the general docs the right place to point at?